### PR TITLE
Removed implied default from strore_(true|false) argparse arguments

### PR DESCRIPTION
**Pull request by jdufresne**
The argparse module automatically sets the default value for store_true
and store_false arguments to its opposite. It is redundant to specify it
explicitly.

See Python documentation for more details:

https://docs.python.org/3/library/argparse.html#action

### CODE REVIEW

File: django/core/management/commands/makemigrations.py [hunk 1]
```diff
@@ -27,20 +27,20 @@ def add_arguments(self, parser):
...
         )
         parser.add_argument(
-            '--empty', action='store_true', dest='empty', default=False,
+            '--empty', action='store_true', dest='empty',
             help="Create an empty migration.",
         )
         parser.add_argument(
             '--noinput', '--no-input',
-            action='store_false', dest='interactive', default=True,
+            action='store_false', dest='interactive',
```
Comment (timgraham) [inline django/core/management/commands/makemigrations.py]: combine with previous line?

File: django/core/management/commands/flush.py [hunk 2]
```diff
@@ -16,7 +16,7 @@ class Command(BaseCommand):
     def add_arguments(self, parser):
         parser.add_argument(
             '--noinput', '--no-input',
-            action='store_false', dest='interactive', default=True,
+            action='store_false', dest='interactive',
```
Comment (timgraham) [inline django/core/management/commands/flush.py]: combine with previous line?

File: django/contrib/gis/management/commands/ogrinspect.py [hunk 3]
```diff
@@ -69,15 +69,15 @@ def add_arguments(self, parser):
         )
         parser.add_argument(
             '--multi-geom', action='store_true',
-            dest='multi_geom', default=False,
+            dest='multi_geom',
```
Comment (timgraham) [inline django/contrib/gis/management/commands/ogrinspect.py]: combine with previous line?

**Comment by jdufresne** [comment]
Combined lines as suggested.

**Pull request merged**